documentation/Get-Bootchart-metrics.md

```diff
@@ -0,0 +1,84 @@
+# Get Bootchart from a Clear Container #
+
+The following steps describe how to get a bootchart from a Clear Containers Container.
```

Comment:
The following steps describe how to get a bootchart from a container running Clear Containers.